src/alpaka/AlpakaCore/radixSort.h

```diff
@@ -166,8 +172,11 @@ namespace cms::alpakatools {
         });
         alpaka::syncBlockThreads(acc);
 
-        if (threadIdxLocal == 0)
+        if (threadIdxLocal == 0) {
           ibs -= sb;
+          // cms-patatrack/pixeltrack-standalone#210
+          alpaka::mem_fence(acc, alpaka::memory_scope::Grid{});
```

Comment:
We should benchmark the changes on an NVIDIA GPU to see if this has any negative impact on the performance there.